src/parser/core/modules/CooldownDowntime.tsx

```diff
@@ -105,7 +105,7 @@ export abstract class CooldownDowntime extends Analyser {
 		when they are available, but do not clip your GCD to use them.</Trans>
 	protected checklistTarget = DEFAULT_CHECKLIST_TARGET
 
-	protected defenseChecklistName = <Trans id="core.cooldownDowntime.use-defense-cds">Use your defensive cooldowns</Trans>
+	protected defenseChecklistName = <Trans id="core.cooldownDowntime.use-defense-cds">Defensive cooldown usage</Trans>
```

Comment:
We typically tell people what to do, rather than what they've done.
So leaving this as "Use your defensive cooldowns" would be fine, IMO.

Reply:
The thought here was that since it's not necessarily a "mistake" that I wasn't sure if it made sense to use the improvement style phrasing anymore